doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -842,6 +842,7 @@ Documentation
   * A section about NFC in the :ref:`app_memory` page.
   * A note in the :ref:`ug_ble_controller` about the usage of the Zephyr LE Controller.
   * Documentation for the :ref:`ug_nrf70`.
+  * A section about Gazell in the :ref:`app_memory` page.
```

Comment:
This entry might fit better under the Application development.

Reply:
Fine to leave it as it is, if you think it would be too high-profile under Application development.